### DISCUSSION TITLE
Parallel jobs and Artifacts Outputs/Inputs

### DISCUSSION THREAD

**Discussion by Tetragramato** [argoproj/argo-workflows#7715] [Q&A]
Hello,

Reading the doc, and can't see how i can make it work properly :

**_I have N jobs in parallel, joining in a job._** 

The N jobs produce output Artifacts, and i want this Artifacts as inputs of my "joining" job 🤔  (i don't know how much jobs i can have in parallels, and how much artifacts can be produced)

I watched exemples on input/output Artifacts, but i can't figure out how i can achieve my goal : have a list of input artifacts in my "joining" job.

I saw this issue : https://github.com/argoproj/argo-workflows/issues/934

But for now i'm on Argo V2.12, and this thread don't really answer my needs i think 🤔 

Thanks for your help ! :)

**Comment by sarabala1979** [comment]
@Tetragramato I would recommend upgrading to the latest. v2.12 is not supported.
you can configure your artifacts location for parallel into the same folder with different  name and final job can have folder  as input  artifact so it will download entire folder for final job
